src/settings-ui/Settings.UI/Controls/SettingsGroup/SettingsGroup.xaml

```diff
@@ -1,14 +1,13 @@
 ﻿<ResourceDictionary
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" 
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:controls="using:Microsoft.PowerToys.Settings.UI.Controls">
 
     <Style TargetType="controls:SettingsGroup">
         <Setter Property="ItemsPanel">
             <Setter.Value>
                 <ItemsPanelTemplate>
-                    <StackPanel Orientation="Vertical"
-                                Spacing="2"/>
+                    <StackPanel Orientation="Vertical" Spacing="2" />
```

Comment:
Shouldn't this be more like 12epx?
https://learn.microsoft.com/windows/apps/design/basics/content-basics#spacing-and-gutters:~:text=12epx%20between%20content%20areas